nginx.conf.template

```diff
@@ -0,0 +1,25 @@
+server {
+    listen 80;
+    server_name 172.20.20.98;
+
+    location / {
+        auth_request /check_user/;
+        proxy_pass http://172.20.20.97:5601;
+        proxy_set_header Cookie $http_cookie;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Authorization $OPENSEARCH_BASIC_TOKEN;
+        proxy_set_header Content-Type "application/json";
+    }
+
+    location = /check_user/ {
+            internal;
+            proxy_pass_request_body off;
+            proxy_set_header Content-Length "";
+            proxy_set_header X-Original-URI $request_uri;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_pass http://172.20.20.13/api/core/users/current_user/;
```

Comment:
Have you worked on most recent develop branches? From what I see services under openIMIS network doesn't have IPs assigned to them right now.

Reply:
It could be reachable on docker level by using 'frontend:80' host like it is done on frontend level. Already checked by me on docker-compose with current changes from develop.